src/app/containers/header/header.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { Store, select } from '@ngrx/store';
 import { CookieService } from 'ngx-cookie';
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
```

Comment:
Doesn't look like `of` is actually used anywhere in the code...?

Reply:
Agreed, removed it now.